recurly_test.go

```diff
@@ -40,7 +44,7 @@ type Scenario struct {
 
 // This method on Scenario gives you a *recurly.Client
 // which implements the testing scenario
-func (s *Scenario) MockHTTPClient() *Client {
+func (s *Scenario) MockHTTPClient(options ...ScenarioOptions) *Client {
```

Comment:
Is there a reason this takes `ScenarioOptions` instead of `ClientOptions`?

Also, since this is only used by tests, let's make the options nullable instead of a splatted array. `options *ClientOptions`